GitHub_Learning/Feature/Onboarding/View/OnboardingCell.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  OnboardingCell.swift
+//  GitHub_Learning
+//
+//  Created by Amin Ashhab on 10/11/2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class OnboardingCell: UICollectionViewCell {
+    
+    static let onboardingCellId = "OnboardingCell"
+    
+    var onboardingPageViewModel: OnboardingPageViewModel? {
+        didSet {
+            pageImageView.image = onboardingPageViewModel?.image
+            pageTitle.text = onboardingPageViewModel?.title
+            pageText.text = onboardingPageViewModel?.text
+        }
+    }
+    
+    private lazy var onboardingCellStack: UIStackView = {
+        let stack = UIStackView(arrangedSubviews: [pageImageView,pageTitle,pageTextStackView])
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.axis = .vertical
+        return stack
+    }()
+    
+    let pageImageView: UIImageView = {
+        let iv = UIImageView()
+        iv.contentMode = .scaleAspectFill
+        iv.translatesAutoresizingMaskIntoConstraints = false
+        iv.backgroundColor = .cyan
+        iv.clipsToBounds = true
+        return iv
+    }()
+    
+    let pageTitle: UILabel = {
+        let title = UILabel()
+        title.font = UIFont.systemFont(ofSize: 25.0, weight: .medium)
+        title.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Musis to delat na kazdem view? Nestaci to jen na tom stacku? Nebo dokonce jen na celle?

Reply:
jj máš pravdu stačí to jen jednou na tom stacku